### ISSUE TITLE
Dash and dock displayed in overview (again) on Ubuntu 17.10

### ISSUE THREAD

**Issue by hackel** [micheleg/dash-to-dock#660]
This seems to be a recurring issue.  (#564, #568, #587)  I've installed Dash to Dock v61 from extensions.gnome.org on a brand new Ubuntu 17.10 install, and am seeing the regular dash underneath the dock on the overview screen.  GNOME shell version 3.26.2.  The most recent bug report indicates that the issue was fixed as of 2017-09-05, which it looks like was included in v61.

Interestingly, I am only having this issue on a brand new Ubuntu install that uses the new Ubuntu Gnome session.  On my other machines that were upgraded to 17.10 (and use the old Gnome Shell session), I am not experiencing the issue.

**Comment by micheleg** [comment]
Hi, I haven't made I new release recently (I'm preparing one now), so I'm not sure if this is somehow fixed in master. Could you test the current master branch?

**Comment by hackel** [comment]
Thanks, looking good with  ae7b86e.

**Comment by hackel** [comment]
Actually, I take it back.  The problem has reappeared.  I haven't changed anything, other than suspending/resuming, so I have no idea what happened.

**Comment by hackel** [comment]
In case anyone else comes across this issue...  I realized that I was logged into the "Ubuntu" session, and so I think Dash to Dock was conflicting with the Ubuntu Dock fork which, as a system extension, *cannot* be disabled.  The solution was to install the gnome-session package and switch to the GNOME Session from the login screen.  At that point, I was able to disable Ubuntu Dock properly, and no longer am experiencing duplicate docks on the overview.
